saleor/graphql/checkout/tests/mutations/test_order_create_from_checkout.py

```diff
@@ -207,6 +207,49 @@ def test_order_from_checkout(
     _recalculate_order_prices_mock.assert_not_called()
 
 
+def test_order_from_checkout_with_transaction(
```

Comment:
I see that you updated the rest of the tests, but I think that it would be good to have the same test as this one but for `channel.automatically_confirm_all_new_orders` is False. (Maybe parametrize can work here).
Also I see that explicit test for checkout with transaction is missing (except the one that just added), so I would propose to split this into three tests:
`test_order_from_checkout_with_transaction` <- the one that handles a scenario when checkout has transaction attached
`test_order_from_checkout_with_unconfirmed_status` <- the one that controls the flag on channel and confirm the expected status
`test_order_from_checkout_with_confirmed_status` <- the one that controls the flag on channel and confirm the expected status

The last two can be probably done with pytest.mark.parametrize

@kadewu  WDYT?

Reply:
Make sense, will do that.